### PR TITLE
125 improve test speed

### LINKED ISSUE #125: Test run very slow over time | time per test increases with amount of tests
__Describe the Bug__

Test run very slow over time | time per test increases with amount of tests.
E.g., the time for the first 100tests is significantly faster than the last 100tests.

__Steps to Reproduce__

1. Run `npm run test`
2. Observe the time per test

__Expected Behavior__

Time per test is roughly the same for the first tests compared to the last tests.

__Environment__

- Host (Browser/Node version), if applicable: Chrome Headless
- OS: Linux

### PR DESCRIPTION
**Pull request by MaxTru**
This increases test speed significantly.

Closes #125

### CODE REVIEW

File: test/spec/provider/camunda-platform/FormDataProps.spec.js [hunk 1]
```diff
@@ -200,6 +200,24 @@ describe('provider/camunda-platform - FormDataProps', function() {
     }));
 
 
+    it('should not display', inject(async function(elementRegistry, selection) {
```
Comment (philippfromme) [inline test/spec/provider/camunda-platform/FormDataProps.spec.js:203]: How is this related to the performance issue?

Comment (MaxTru) [reply test/spec/provider/camunda-platform/FormDataProps.spec.js:203]: * So the test was testing that formProps should be displayed in an StartEvent in a subProcess.
  * IMO this is wrong per-se
* However, this test still passed in the past. I guess because the missing clean-up (see performance optimization) somehow led to test being not executed correctly.
* Since the error in the test seems obvious to me (am I wrong) I just corrected it.

Comment (philippfromme) [reply test/spec/provider/camunda-platform/FormDataProps.spec.js:203]: Okay, understood. Thanks for explaining. 🙏🏻

**Review by philippfromme — APPROVED** [review]
Great job! It's lightning-fast again! ⚡

**Comment by barmac** [comment]
Huge ❤️  for this PR.
